### ISSUE TITLE
GTag for Enhanced Ecommerce needs decimal values

### ISSUE THREAD

**Issue by EddyP23** [angulartics/angulartics2#346]
**I'm submitting a bug report**.

**TL;DR**  `Angulartics2GoogleGlobalSiteTag.eventTrack` rounds value field to an integer. It might be the correct behaviour in most cases, but not in all.

**Elaborate version**

We are using Google Analytics Enhanced ECommerce via GTag and we want to send various actions, like `purchase`, `refund`, etc. One of the parameters that we send together is the value of the purchase/refund, which is decimal, not integer, e.g. `USD 3.76`.

We do this with the following code:

```
    this.angulartics2GoogleGlobalSiteTag.eventTrack('purchase', {
      gstCustom: {
        transaction_id: 'tran_id',
        value: 3.73,
        currency: 'USD',
        items: [
          {
              ...
          }
        ]
      }
    });
```

The problem is, that `Angulartics2GoogleGlobalSiteTag.eventTrack` method calls a private `Angulartics2GoogleGlobalSiteTag.eventTrackInternal` which calls ` cleanProperties(properties)`  which does the following:

```
  private cleanProperties(properties: { [key: string]: any }): void {
    // GA requires that eventValue be an non-negative integer, see:
    // https://developers.google.com/analytics/devguides/collection/gtagjs/events
    if (properties.value) {
      const parsed = parseInt(properties.value, 10);
      properties.value = isNaN(parsed) ? 0 : parsed;
    }
  }
```

The assumption in the comment might be correct for ordinary events, but ECommerce explicitly needs decimal values, see [Google ECommerce documentation here](https://developers.google.com/analytics/devguides/collection/gtagjs/enhanced-ecommerce#measure_purchases).

This is clearly wrong, as our Google Analytics now shows integer values for all purchases.

**Hacky workaround**
Currently, the only hacky fix that I can think of is the following:

```
    this.angulartics2GoogleGlobalSiteTag['cleanProperties'] = () => {};
```

**Expected behaviour**
Clearly, it is not to round the `value` parameter in all cases. I don't know enough about Google Analytics to suggest a solution though.
